packages/core/test/animation/animation_query_integration_spec.ts

```diff
@@ -3302,6 +3302,63 @@ describe('animation query tests', function() {
            expect(p2.duration).toEqual(1000);
            expect(p2.element.classList.contains('parent')).toBeTrue();
          });
+
+      it('should disable the animation for the element that they are disabled on even if a parent animation query and animate sub animations',
```

Comment:
This is a confusing sentence that makes it hard to follow the test's intent. I think this option is clearer.
```suggestion
      it('should disable the animation for the given element regardless of existing parent element animation queries or child element animation queries',
```

Reply:
Thanks, have fixed.